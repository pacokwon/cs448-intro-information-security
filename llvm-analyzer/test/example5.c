#include "homework.h"

int main() {
  int x = source();
  int y;
  if (x > 0) {
    y = sanitizer(x);
    sink(y); // safe
  } else {
    y = x;
  }
  sink(y); // bug
  return 0;
}
