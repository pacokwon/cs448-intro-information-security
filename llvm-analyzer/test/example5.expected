Potential Tainted-flow @ example5.c:main:12:3 (call void @sink(i32 %y.0), !dbg !23)
